lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -554,19 +554,35 @@ struct RvalueExprVisitor {
         // Handle expressions.
         if (!listExpr->type->isSimpleBitVector()) {
           if (listExpr->type->isUnpackedArray()) {
+            if (listExpr->type->isFixedSize()) {
+              const auto &uaType =
+                  listExpr->type->as<slang::ast::FixedSizeUnpackedArrayType>();
+              auto value = context.convertRvalueExpression(*listExpr);
+              if (!value)
+                return {};
+              context.collectConditionsForUnpackedArray(uaType, value,
+                                                        conditions, lhs, loc);
+              cond = conditions.back();
+              conditions
+                  .pop_back(); // avoiding repetition of cond in the vector
+            } else {
+              mlir::emitError(loc, "unsized unpacked arrays in 'inside' "
```

Comment:
When you do the LogicalResult conversion, you can just `return mlir::emitError`.

It might be useful to emit the error on the op?  I'm not sure, I haven't thought about it.